charts/pmm-ha/templates/pg-secret-init-job.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: secret-generator
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: secret-manager
+  namespace: {{ .Release.Namespace }}
+rules:
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["create", "get", "update", "patch"]
+- apiGroups: ["apps"]
+  resources: ["statefulsets"]
+  verbs: ["get", "patch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: secret-generator-binding
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: secret-manager
+subjects:
+- kind: ServiceAccount
+  name: secret-generator
+  namespace: {{ .Release.Namespace }}
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: secret-init-job
+  namespace: {{ .Release.Namespace }}
+spec:
+  ttlSecondsAfterFinished: 86400  # Clean up after 24 hours
+  template:
+    spec:
+      serviceAccountName: secret-generator
+      restartPolicy: OnFailure
+      initContainers:
+      - name: generator
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+        command:
+        - /bin/bash
+        - -c
+        - |
+          set -e
+          echo "Generating cryptographic key..."
+
+          # Generate the secret using pmm-encryption-rotation and write to shared volume
+          /usr/sbin/pmm-encryption-rotation --generate-key > /shared/secret-value
+          
+          echo "Key generated successfully"
+        volumeMounts:
+        - name: shared-data
+          mountPath: /shared
+      containers:
+      - name: kubectl-applier
+        image: bitnami/kubectl:latest
```

Comment:
Bitnami should probably not be reintroduced

Reply:
Do you have a good alternative in mind?